daemon/container_unix.go

```diff
@@ -1358,11 +1358,16 @@ func (daemon *Daemon) setupIpcDirs(container *Container) error {
 			return err
 		}
 
-		// When ShmSize is 0 or less, the SHM size is set to 64MB.
-		if container.hostConfig.ShmSize <= 0 {
-			container.hostConfig.ShmSize = 67108864
+		// this is to handle an omitted HostConfig passed to the API which won't hit
+		// adaptContainerSettings in daemon_unix.go and won't have a default ShmSize
```

Comment:
can you add back a mention that we're setting it to `64MB` by default? prevents people from scratching their heads what the magic number `67108864` is :smile: